clients/src/main/java/org/apache/kafka/common/security/authenticator/SaslServerAuthenticator.java

```diff
@@ -73,6 +73,7 @@
 
 public class SaslServerAuthenticator implements Authenticator {
 
+    private static final int MAX_RECEIVE_SIZE = 131072; // Chosen as twice the max receive size for GSSAPI
```

Comment:
Not sure. This limit is meant to catch malicious behaviour. What would be the benefit of adding it to the client?